rolling/distribution.yaml

```diff
@@ -3184,6 +3184,22 @@ repositories:
       url: https://github.com/ros2/variants.git
       version: master
     status: maintained
+  vision_msgs:
+    doc:
+      type: git
+      url: https://github.com/ros-perception/vision_msgs.git
+      version: noetic-devel
```

Comment:
Good catch on the branch. I'll update to `ros2`

The repository has changed ownership to ros-perception since the foxy release.

Reply:
> The repository has changed ownership to ros-perception since the foxy release.

Thanks, that is good to know.

Should we also move the release repository into https://github.com/ros2-gbp?

Reply:
Will moving the repository cause issues if we want to do any more ROS1 releases (it uses the same repo). Maybe it would be better to make a new release repo in ros2-gbp specifically for the ROS2 releases, keeping ROS1 releases at the old location

Reply:
> Will moving the repository cause issues if we want to do any more ROS1 releases (it uses the same repo). Maybe it would be better to make a new release repo in ros2-gbp specifically for the ROS2 releases, keeping ROS1 releases at the old location

Thanks, I didn't realize that.

If we keep the release repository at https://github.com/Kukanani/vision_msgs-release.git , then you can continue to use it for both ROS 1 and ROS 2.  The downside to that is that we won't be able to automatically release vision_msgs into new stable ROS 2 distributions as they come along.

Putting the release repository into ros2-gbp means that we will be able to release the package automatically into new stable ROS 2 distributions.

At least, that is my understanding.  Pinging @nuclearsandwich and @cottsay to give advice (since they just did the branching for Galactic).

Reply:
I'm going to go ahead with this one; if we need to change release repositories, we can always do it later.

Reply:
@Kukanani @clalancette Apologies for the late response, still trying to optimize the GitHub notification workflow so I'm not buried but not missing the good stuff.



> Will moving the repository cause issues if we want to do any more ROS1 releases (it uses the same repo). Maybe it would be better to make a new release repo in ros2-gbp specifically for the ROS2 releases, keeping ROS1 releases at the old location

Moving the release repository to ros2-gbp won't create any technical problems. It may cause some confusion but there's nothing stopping us from doing so.



> If we keep the release repository at [Kukanani/vision_msgs-release.git](https://github.com/Kukanani/vision_msgs-release.git) , then you can continue to use it for both ROS 1 and ROS 2. The downside to that is that we won't be able to automatically release vision_msgs into new stable ROS 2 distributions as they come along.

This is close, any release repositories in rolling that _are not_ in the ros2-gbp org will be copied there for the next roll (the next roll will be from `rolling` on 20.04 into `rolling` on 22.04 or `rolling` on 20.04 into `rolling` on 20.04 and Debian bullseye).
These new repositories will then need to be reconciled with their out-of-organization counterparts so I would consider it a soft requirement that will slowly harden in time.

My recommendations based on the above offer two paths:

1. Move the repository to ros2-gbp and use that repository for all releases, ROS and ROS 2 alike.
2. Continue with ROS 1 releases with the repository as-is and move the ROS 2 releases to a new copy in ros2-gbp.

To me, option 1 makes the most sense as it is the simplest option but I think option 2 is a reasonable compromise against raised eyebrows at seeing ros2-gbp repositories in noetic.

Reply:
@nuclearsandwich Let's move it to `ros2-gbp` (option 1). Looks like I need repository creation access in that org, or I need to add someone else who does to my repo so they can initiate the transfer.

Reply:
@nuclearsandwich any update on this? Happy to do whatever would make the transfer to `ros2-gbp` easier

Reply:
@Kukanani if I understand the latest GitHub docs correctly it may not be possible for an external collaborator with admin rights to transfer repositories. However there are some outstanding issues that make giving you repository creation access to the ros2-gbp org challenging so let's try adding me as an external collaborator on https://github.com/Kukanani/vision_msgs-release.git and I'll try to transfer it. If that doesn't work then we'll do what we need to do to add you to the ros2-gbp organization.

Reply:
@nuclearsandwich Added you as a collaborator as requested above.

Reply:
Well as feared that didn't give me the necessary permissions so I've sent you an invitation to the vision_msgs team within ros2-gbp. Once you're on the team you should be able to transfer the repository in yourself following this guide https://docs.github.com/en/github/administering-a-repository/managing-repository-settings/transferring-a-repository#transferring-a-repository-owned-by-your-user-account

Once you've done so I'll do a little bit to adjust the permissions and get you set up. Thanks for working this one out with us.

Reply:
Just a heads up @Kukanani that I adjusted the permissions so that you have access via a team rather than directly. If that gives you any trouble please create an issue on ros2-gbp/vision_msgs-release and @ me.